components/SSI/AddFunds/index.tsx

```diff
@@ -529,15 +529,32 @@ function Component(props: InputType) {
                 <div className={styles.originatorInfoWrapper}>
                   <p className={styles.originatorType}>Zilpay wallet:&nbsp;</p>
                   <p className={styles.originatorAddr}>
-                    {loginInfo.zilAddr?.bech32}
+                    <a
+                      style={{ textTransform: "lowercase", color: "#ffff32" }}
+                      href={`https://devex.zilliqa.com/address/${
+                        loginInfo.zilAddr?.bech32
+                      }?network=https%3A%2F%2F${
+                        net === "mainnet" ? "" : "dev-"
+                      }api.zilliqa.com`}
+                      rel="noreferrer"
+                      target="_blank"
+                    >
+                      {loginInfo.zilAddr?.bech32}
+                    </a>
                   </p>
                 </div>
               ) : (
                 <>
                   {originator_address.username === undefined && (
                     <p style={{ marginBottom: "10%" }}>
                       About to send funds from{" "}
-                      {zcrypto.toBech32Address(originator_address?.value)}
+                      {zcrypto.toBech32Address(originator_address?.value)}{" "}
+                      into&nbsp;
+                      <span style={{ color: "#ffff32" }}>
+                        {loginInfo?.username
+                          ? `${loginInfo?.username}.did`
+                          : zcrypto.toBech32Address(loginInfo?.address)}{" "}
```

Comment:
this situation should never happen atm

Reply:
Then, you want me to put only `zcrypto.toBech32Address(loginInfo?.address)` ?

Reply:
I mean that there's always gonna be a username logged in when adding funds from /balance. You can leave it as it is for now.

Reply:
Noted